manim/mobject/svg/text_mobject.py

```diff
@@ -29,6 +50,16 @@ def __init__(self, start, end, font, slant, weight, line_num=-1):
         self.line_num = line_num
 
 
+"""
+Text is VGroup() of each characters
```

Comment:
Shouldn't this be in `Text`'s Docstring?

Reply:
I did not add docstring for Text because I don't know exactly for what some parameters are used and I also don't able to understand some methods deeply.

Reply:
Let's work on this on another PR